docs/business-finder-email-signup.md

```diff
@@ -0,0 +1,15 @@
+# EU Exit Readiness Business Finder Email signup
+
+Currently only the [EU Exit Business Readiness Finder](https://www.gov.uk/find-eu-exit-guidance-business) is making use of these features, but other finders could use these in the future.
+
+## Subscribing to multiple facet values
+
+Users can [pick and choose which `facet_values` ('links') to subscribe to](https://github.com/alphagov/finder-frontend/pull/1191), and will be emailed when content is tagged with _any of the links_ (provided the 'send email' checkbox is checked in content tagger). When already tagged content is updated, only major updates will trigger emails (as per normal email subscriptions).
+
+So, if a user has subscribed to updates about `Aerospace` or `EU Citizens`, and a piece of content linked only to `EU Citizens` is updated, an email will be sent. This functionality is described in the [email-alert-api documentation](https://github.com/alphagov/email-alert-api/blob/master/doc/matching-content-to-subscriber-lists.md).
+
+The titles for email subscriber lists are usually generated by concatenating the list of all of the facets the user has selected. This is fine for finders where users can only subscribe to a few facets, but in the business finder this list could get very long. To make the subscription title easier to read, the business finder creates title with a [comma-separated list of facets](https://github.com/alphagov/finder-frontend/pull/email_alert_list_title_builder.rb).
+
+## Changing the default selected frequency
+
+There is a [`default_frequency` parameter](https://github.com/alphagov/finder-frontend/pull/1104) which overrides the default selected frequency from "Immediately" to "Daily" in the EU Exit Business Readiness Finder. See the [corresponding PR in email-alert-frontend](https://github.com/alphagov/email-alert-frontend/pull/461). 
```

Comment:
Rather than linking to the whole PR, can you link to the code, or the [commit](https://github.com/alphagov/finder-frontend/commit/557a242c599b397342594863c0b00e0f9eb70775) or interest?

Same with email-alert-api, can you link to the code, rather than the PR: https://github.com/alphagov/email-alert-frontend/blob/master/app/controllers/subscriptions_controller.rb

Reply:
I'm not sure about linking to the current version of the code, which after all is only a snapshot of where we are right now. If the code gets refactored out into another file in future, it's probably going to confuse people more than if we'd linked to the original PR (which stays constant).

The PR is also quite small and it could be useful to show the other parts of the codebase that the change touches, and the accompanying tests that describe the behaviour nicely.

Happier to link to the commit than to the current state of the code, as that stays constant.